sites/website/src/docs/resources/cheat-sheet.md

```diff
@@ -0,0 +1,695 @@
+* Use this library when you want to implement something like Google's Material Design or Twitter Bootstrap.
+
+```ts
+import {
+  Button as FoundationButton,
+  buttonTemplate as template,
+} from "@microsoft/fast-foundation";
+import { css } from "@microsoft/fast-element";
+
+export class MyButton extends FoundationButton {
+  ...
+}
+
+export const buttonStyles = css`
+  :host {
+    background-color: azure;
+  }
+`;
+
+export const myButton = MyButton.compose({
+  baseName: "button",
+  baseClass: FoundationButton,
+  template,
+  styles: buttonStyles,
+  shadowOptions: {
+    delegatesFocus: true,
+  },
+});
+```
+
+---
+
+### [@microsoft/fast-components](https://www.fast.design/docs/components/getting-started)     
+
+**A component library that implements Microsoft's [FAST Frame Design System](https://www.fast.design/docs/design-systems/fast-frame/).**
+
+* Assembles the building blocks of `@microsoft/fast-foundation` to create its component set. 
+* Use this library when you want to integrate [FAST Components](#using-components) into an existing site or app.
+
+To install the components, use either `npm` or `yarn`:
+
+```shell
+npm install --save @microsoft/fast-components
+``` 
+
+```shell
+yarn add @microsoft/fast-components
+```
+
+To use a Web Component as a custom element in HTML, the custom element must be registered:
+
+```ts
+import { 
+  fastButton, 
+  provideFASTDesignSystem 
+} from "@microsoft/fast-components";
+
+provideFASTDesignSystem()
+  .register(
+    fastButton() // custom element
+  );
+```
+
+---
+### [@fluentui/web-components](https://docs.microsoft.com/en-us/fluent-ui/web-components/)
+
+**A component library that implements Microsoft's [Fluent Design System](https://www.microsoft.com/design/fluent/#/).**
+
+* Assembles the building blocks of `@microsoft/fast-foundation` to create its component set. 
+* Use this library when you want components to look and feel like those found in Microsoft products.
+
+To install the components, use either `npm` or `yarn`:
+
+```shell
+npm install --save @fluentui/web-components
+``` 
+
+```shell
+yarn add @fluentui/web-components
+```
+
+To use a Web Component as a custom element in HTML, the custom element must be registered:
+
+```ts
+import { 
+  fluentButton, 
+  provideFluentDesignSystem 
+} from "@fluentui/web-components";
+
+provideFluentDesignSystem()
+  .register(
+    fluentButton() // custom element
+  );
+```
+
+---
+
+If you are attempting to configure the components for integration into a specific Microsoft product, the following table provides `AccentBaseColor` values you can use:
+
+| Product     | AccentBaseColor | 
+| :---        | :---            | 
+| Office      | #D83B01         | 
+| Word        | #185ABD         | 
+| Excel       | #107C41         | 
+| PowerPoint  | #C43E1C         | 
+| Teams       | #6264A7         | 
+| OneNote     | #7719AA         | 
+| SharePoint  | #03787C         | 
+| Stream      | #BC1948         | 
+
+---
```

Comment:
@EisenbergEffect @huynhicode I think we should remove this - while we're the primary maintainers of Fluent UI Web Components as well, there is an entire documentation strategy there and the code _doesn't_ live here. I think adding this here is going to cause more work for us in managing where issues should be raised and where support should come from. 

My suggestion is that if we want to add this, we should add it to Fluent. Alterntively, we could have a "projects" portion where we call out _all_ partners and provide more context. As it stands right now, this is an implementation we just happen to be more involved with than not, but it's not part of FAST - it's an implementation of it. It's especially odd as we jump back into FAST below.

Reply:
So, maybe remove the chart and the accent color note above that an dreplace it with a link to the Fluent UI Web Components documentation?

@Chris Do you have the repo link where I could contribute this over there? How are we managing that now?

Reply:
Fork and contribute to our WC Readme is one option, we can also add to the msdocs site: https://github.com/microsoft/fluentui

Reply:
I was thinking to add it to docs.microsoft.com docs, in the styling section.

Reply:
I think that's probably a better place for the last part. The rest of this should be represented in the docs there.

Reply:
@huynhicode just for clarity, this is really the primary thing I'm blocking on. Once this is removed I'm happy to approve given the other changes being made :)

Thanks for this!

Reply:
@chrisdholt Make sense, thank you!  I appreciate all the feedback!

Reply:
@chrisdholt @EisenbergEffect Thank you - updated!  I removed the table and corresponding table note, and added a link to Fluent UI Web.